platform/ios/platform/ios/app/Info.plist

```diff
@@ -26,12 +26,28 @@
 	<true/>
 	<key>NSHumanReadableCopyright</key>
 	<string>© 2014–2019 Mapbox</string>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>The map will display your location. If you choose Always, the map may also use your location when it isn’t visible in order to improve OpenStreetMap and Mapbox products.</string>
```

Comment:
@petr-pokorny-1  - Potential change:   

* Remove the section ~~ and Mapbox products~~. 
* I don't even think changing to MapLibre would make sense

Reply:
text updated